### DISCUSSION TITLE
Sort table by descending years

### DISCUSSION THREAD

**Discussion by LucyGwilliamAdmin** [open-sdg/open-sdg#1399] [Ideas]
Suggested by @WohnungNDS

**Describe the solution that you need**
On the table tab, years should be descending instead of acsending by default

**Why do you need this feature?**
Users will see most recent data first

![image](https://user-images.githubusercontent.com/43404358/139831958-7eadbe81-74dd-479d-a503-48fe0a78d192.png)

**Comment by LucyGwilliamAdmin** [comment]
@phillipgardiner what are your initial thoughts on this from a usability point of view?

**Reply by phillipgardiner** [reply]
@LucyGwilliamAdmin The fact users are able to change the order by use of the arrow is a positive. Whether the default is ascending or descending very much comes down to the task of the user and which data users are most likely to be looking for (e.g. are they looking for recent data or older data). If recent data then it makes sense to show it descending, if older data it makes sense to show it ascending (this becomes more relevant on mobiles etc where scrolling may be required). Unfortunately we don't have any research that shows the most likely scenario (although my instinct based on observations would be descending). Therefore it seems a configuration option may be the way to go so each country can choose their own order based on what they know about thier users needs.

**Comment by WohnungNDS** [comment]
To my mind it must not be **by default**, but may be we can adjust it site wide...

**Reply by LucyGwilliamAdmin** [reply]
Yes, possibly a configuration option
